src/components/expression/expressionComparisonRibbon.js

```diff
@@ -110,7 +110,7 @@ class ExpressionComparisonRibbon extends React.Component {
                     geneId={geneId}
                     groups={groups}
                     onClick={this.handleBlockClick}
-                    overrideColor={geneTaxon === TAXON_IDS.HUMAN && '#dedede'}
+                    overrideColor={geneTaxon === TAXON_IDS.HUMAN && '#dedede' ? 't' : 'f'}
```

Comment:
This isn't the right way to fix the proptypes warning and would actually introduce a bug in the expression ribbon. Please remove.